third_party/tsl/tsl/platform/float8.h

```diff
@@ -36,7 +36,9 @@ namespace float8_internal {
 // Forward-declarations of classes.
 class float8_e4m3fn;
 class float8_e4m3b11;
+class float8_e4m3fnuz;
```

Comment:
This whole file is actually obsolete.  We're transitioning to use a new package [`ml_dtypes`](https://github.com/jax-ml/ml_dtypes), where we've already added implementations for `float8_e4m3fnuz`/`float8_e5m2fnuz` [[source](https://github.com/jax-ml/ml_dtypes/blob/main/ml_dtypes/include/float8.h)].

Reply:
Thanks @cantonios. It was [me](https://github.com/jax-ml/ml_dtypes/pull/57) that added them to ml_dtypes too. 😂
I'm happy to drop these tsl changes, if that transition is imminent.